vllm/v1/spec_decode/eagle.py

```diff
@@ -319,8 +319,10 @@ def load_model(self, target_model: nn.Module) -> None:
         target_attn_layer_names = set(
             get_layers_from_vllm_config(self.vllm_config, Attention).keys())
 
-        self.model = get_model(vllm_config=self.vllm_config,
-                               model_config=draft_model_config)
+        from vllm.compilation.backends import set_model_tag
+        with set_model_tag("eagle_head"):
```

Comment:
nit: could we name this something like "set_compile_region"? That would make it clearer that this is 1:1 with a fullgraph torch.compile region